src/core/network/networkGraph.ts

```diff
@@ -751,98 +758,25 @@ export class NetworkGraph {
-     * in the connector: */
-    const connectorPlusSymbolComponents = [
-      {
-        name: 'plus-hline-white',
-        // coordinates with current config: x1: 30, y1: 25.5, x2: 39.5, y2: 25.5
-        x1: connectorEndPos.x + this._connectorRadius / 3,
-        y1: connectorEndPos.y - (13 / 12) * this._connectorRadius,
-        x2: connectorEndPos.x + (23 / 12) * this._connectorRadius,
-        y2: connectorEndPos.y - (13 / 12) * this._connectorRadius,
-      },
-      {
-        name: 'plus-vline-white',
-        // coordinates with current config: x1: 35, y1: 21.5, x2: 35, y2: 31
-        x1: connectorEndPos.x + (7 / 6) * this._connectorRadius,
-        y1: connectorEndPos.y - (21 / 12) * this._connectorRadius,
-        x2: connectorEndPos.x + (7 / 6) * this._connectorRadius,
-        y2: connectorEndPos.y - (1 / 6) * this._connectorRadius,
-      },
-      {
-        name: 'plus-hline-colored',
-        // coordinates with current config: x1: 31.5, y1: 25.5, x2: 38.5, y2: 25.5
-        x1: connectorEndPos.x + (7 / 12) * this._connectorRadius,
-        y1: connectorEndPos.y - (13 / 12) * this._connectorRadius,
-        x2: connectorEndPos.x + (21 / 12) * this._connectorRadius,
-        y2: connectorEndPos.y - (13 / 12) * this._connectorRadius,
-      },
-      {
-        name: 'plus-vline-colored',
-        // coordinates with current config: x1: 35, y1: 22, x2: 35, y2: 29.5
-        x1: connectorEndPos.x + (7 / 6) * this._connectorRadius,
-        y1: connectorEndPos.y - (5 / 3) * this._connectorRadius,
-        x2: connectorEndPos.x + (7 / 6) * this._connectorRadius,
-        y2: connectorEndPos.y - (5 / 12) * this._connectorRadius,
-      },
-    ];
-
-    let lines = connector.selectAll('line');
-
-    const correctState = !this._state.enableConnection && !this.state.dragging;
-    for (let line of connectorPlusSymbolComponents) {
-      /* select all lines with the correct name (in the whole d3js graphics)
-       * and alter only the coordinates of those whose node is selected/focused */
-      lines
-        .filter(function () {
-          return d3.select(this).attr('name').indexOf(line.name) >= 0;
-        })
-        .transition(t)
-        .attr('x1', (node: Node) =>
-          (node.view.isFocused() || node.view.isSelected()) && correctState
-            ? line.x1
-            : 0
-        )
-        .attr('y1', (node: Node) =>
-          (node.view.isFocused() || node.view.isSelected()) && correctState
-            ? line.y1
-            : 0
-        )
-        .attr('x2', (node: Node) =>
-          (node.view.isFocused() || node.view.isSelected()) && correctState
-            ? line.x2
-            : 0
-        )
-        .attr('y2', (node: Node) =>
-          (node.view.isFocused() || node.view.isSelected()) && correctState
-            ? line.y2
-            : 0
-        );
-    }
+    connector
+      .selectAll('line')
+      .transition(t)
+      .attr('opacity', (node: Node) => (node.view.isFocused() ? 1 : 0));
```

Comment:
In that case, the plus symbol disappears, when a node is selected, but out of focus, so that the visual indication of the plus symbol is gone. What do you think about:
```suggestion
      .attr('opacity', (node: Node) =>
        node.view.isFocused() || node.view.isSelected() ? 1 : 0
      );
```

Reply:
It was an experiment. You can decide which way would be better.

Do you think, a persistent appearance of plus is sufficient?

Reply:
I think it would be a bit more intuitive if we keep the connector as a whole - and thus have a persistent appearance of the plus symbol. Anyways, we could discuss it with @bweyers. I added a comment, if you think that this is ok.